homeassistant/components/mychevy.py

```diff
@@ -33,10 +33,12 @@
 MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=30)
 ERROR_SLEEP_TIME = timedelta(minutes=30)
 
+CONF_COUNTRY = 'country'
 CONFIG_SCHEMA = vol.Schema({
     DOMAIN: vol.Schema({
         vol.Required(CONF_USERNAME): cv.string,
         vol.Required(CONF_PASSWORD): cv.string,
+        vol.Optional(CONF_COUNTRY, default="us"): cv.string
```

Comment:
`vol.All(cv.string, vol.In(['us', 'ca']))`?

Reply:
Thanks! I was wondering how to do that.